### PR TITLE
Add localisation to Section Editor

### PR DESCRIPTION
**Pull request by rbarbour**
Adds localisation to the Section Editor so all strings now managed through locale files.
Adds locale / language selector to the Section Editor UI.

### CODE REVIEW

**Review by maeda-m — APPROVED** [review]
Thanks for the great idea.

There are some improvements in the Japanese translation, but I think the rest is fine.

**Review by hidakatsuya — COMMENTED** [review]
Thanks for the great work! I have confirmed that it works perfectly.

However, there seems to be a few improvements in the translation content and code style. Could you please fix them?

Also, could you please rebase to master once to remove the merge commit 2d630908ea81908b4aa95d14f562645d2592ed6c?

File: section-report/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "lint": "vue-cli-service lint",
     "electron:build": "cross-env VUE_APP_HANDLERS=electron vue-cli-service electron:build",
     "electron:serve": "cross-env VUE_APP_HANDLERS=electron vue-cli-service electron:serve",
+    "i18n:report": "vue-cli-service i18n:report --src \"./src/**/*.?(js|vue)\" --locales \"./src/locales/**/*.json\"",
```
[the remark this answers is not in the document]
Comment (rbarbour) [reply section-report/package.json:12]: No problem, glad to help.
Hopefully I've resolved your main comments, but please let me know if there's anything else I missed.
Cheers,
R

**Review by hidakatsuya — APPROVED** [review]
LGTM!

Thank you very much!

**Comment by hidakatsuya** [comment]
I've added it to [CHANGELOG.md](https://github.com/thinreports/thinreports-editor/blob/master/section-report/CHANGELOG.md).

**Pull request merged**